tools/graphics/cat_prey_blender.py

```
@@ -0,0 +1,245 @@
+    capability = "prey_animation_rendering"
+    provider = "blender"
+    stability = ToolStability.BETA
+    execution_mode = ExecutionMode.SYNC
+    determinism = Determinism.SEEDED
+    runtime = ToolRuntime.LOCAL_GPU
+    dependencies: list[str] = []
+    install_instructions = "Install Blender 4.5+ or configure BLENDER_PATH as documented by blender_world."
+    capabilities = [
+        "apply_prey_motion", "render_prey_animation", "ground_following_motion",
+        "procedural_body_micro_motion", "procedural_forest_floor", "resumable_png_render",
+    ]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)